### ISSUE TITLE
Need a way to change databases after connection is made

### ISSUE THREAD

**Issue by gbenedict** [psequel/psequel#83]
Currently, you have to know the name of the database prior to connecting. Ideally you should be able to  move between databases after connecting to any database (including the default postgres).

**Comment by houshuang** [comment]
Agree, would be great to see a list of databases and quickly jump between them, similar to how it's done in Sequel.app.

**Comment by fidelisrafael** [comment]
This will be great! PgAdmin do this too! (I suggested to add a 'DB Restriction' feature too, cuz this can be a huge problem when connecting to database in Heroku for example)

![](http://i.imgur.com/bnDWB7j.png)

**Comment by psequel** [comment]
That's a great suggestion! I actually added a dropdown for selecting database in an unreleased version and found that it's killing Heroku, so I decided to remove that feature.

**Comment by kyle-mccarthy** [comment]
+1 For this, SequelPro will allow you to connect to a host without listing the DB name and then it gives a list to choose from.

**Comment by robertoschwald** [comment]
+1
